website/tutorial/basic/12-summary.mdx

```diff
@@ -0,0 +1,11 @@
+---
+id: 11-summary
+title: Summary
+sidebar_label: Summary
+---
+
+In this tutorial, you learned how to build a GraphQL server from scratch using graphql-yoga.
+
+The resolvers of your GraphQL server are implemented using Prisma Client which is responsible for database access.
+
```

Comment:
Link to the advanced tutorial!
Link to read about The Guild - https://www.the-guild.dev/about-us
Maybe link to Discord/Twitter/other things?

Reply:
we dont have the advanced tutorial yet, right?

Reply:
done!